src/pip/_internal/vcs/subversion.py

```diff
@@ -53,20 +69,63 @@ def get_vcs_version(self):
         version_prefix = 'svn, version '
         version = self.run_command(['--version'], show_stdout=False)
         if not version.startswith(version_prefix):
-            return None
+            return ()
 
         version = version[len(version_prefix):].split()[0]
         version_list = version.split('.')
         try:
             parsed_version = tuple(map(int, version_list))
         except ValueError:
-            return None
-
-        if not parsed_version:
-            return None
+            return ()
 
         return parsed_version
 
+    def get_vcs_version(self):
+        # type: () -> Tuple[int, ...]
+        if self._vcs_version is not None:
+            # Use cached version, if available.
+            # If parsing the version failed previously (empty tuple),
+            # do not attempt to parse it again.
+            return self._vcs_version
+        else:
+            vcs_version = self.call_vcs_version()
+            self._vcs_version = vcs_version
+            return vcs_version
+
+    def get_vcs_global_options(self):
```

Comment:
Since not _every_ command needs these options, maybe a more specific name would be better (e.g. `get_remote_call_options()` or `get_server_call_options()`)?